ESPWebThingAdapter.h

```diff
@@ -118,6 +152,66 @@ class WebThingAdapter {
     return false;
   }
 
+  #ifndef WITHOUT_WS
+  void sendErrorMsg(DynamicJsonBuffer &buffer, AsyncWebSocketClient& client, int status, const char* msg) {
```

Comment:
The spec does not specify how to respond on a bad request (eg json parse error, missing field etc).
I'm sending back an object with "error" (string) and "status" (http code) for now.